package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "npm run prisma:generate && npm run migrate:postgres -y && NODE_ENV=production next build",
+    "build": "npm run prisma:generate && NODE_ENV=production next build",
```

Comment:
auto migration on production DB made us exceed some free limit and thats what made all previous deployments failed. i fixed it so migrations on production db will need to run manually to avoid it